app/routes.py

```diff
@@ -0,0 +1,357 @@
+from datetime import datetime, timedelta
+from app.models.video import Video
+from app.models.customer import Customer
+from app.models.rental import Rental
+
+
+videos_bp = Blueprint("videos", __name__, url_prefix=("/videos"))
+
+#POST/CREATE 
+@videos_bp.route("",methods=["POST"])
+def post_videos():
+    request_body = request.get_json()
+    if "title" not in request_body:
+        return{
+            "details": "Request body must include title."
+        }, 400
+    
+    elif "release_date" not in request_body:
+        return{
+            "details": "Request body must include release_date."
+        }, 400
+    
+    elif "total_inventory" not in request_body:
+        return{
+            "details": "Request body must include total_inventory."
+        }, 400
+
+    new_video = Video(
+        title = request_body["title"], 
+        release_date = request_body["release_date"],
+        total_inventory = request_body["total_inventory"]
+    )
+    db.session.add(new_video)
+    db.session.commit()
+
+    return jsonify(new_video.to_video_object()), 201
+
+#GET
+@videos_bp.route("", methods=["GET"])
+def get_videos():
+    videos = Video.query.all()
+    videos_response = []
+    for video in videos:
+        videos_response.append(video.to_video_object())
+    return jsonify(videos_response), 200
+
+@videos_bp.route("/<video_id>",methods=["GET"])
+def get_video(video_id):
+    if video_id.isnumeric() != True:
+        return {"message": "Video id provided is not a number."}, 400
+    
+    video = Video.query.get(video_id)
+
+    if video is None:
+        return {"message": f"Video {video_id} was not found"}, 404
+        
+
+    return{
+        "id": video.id,
+        "title": video.title,
+        "release_date": datetime.now(),
+        "total_inventory": video.total_inventory
+    }
+#PUT
+@videos_bp.route("/<video_id>",methods=["PUT"])
+def update_video(video_id):
+    video = Video.query.get(video_id)
+    request_body = request.get_json()
+
+    if video is None:
+        return {"message": f"Video {video_id} was not found"}, 404
+
+    if "title" not in request_body or "release_date" not in request_body or "total_inventory" not in request_body:
+        return {
+            "details": "Invalid data"
+        }, 400
+
+    video.title = request_body["title"]
+    video.release_date = request_body["release_date"]
+    video.total_inventory = request_body["total_inventory"]
+
+    db.session.commit()
+
+    return {
+        "id": video.id,
+        "title": video.title,
+        "release_date": datetime.now(),
+        "total_inventory": video.total_inventory
+    }
+#DELETE
+@videos_bp.route("/<video_id>", methods=["DELETE"])
+def delete_video(video_id):
+
+    video = Video.query.get(video_id)
+
+    if video is None:
+        return {"message": f"Video {video_id} was not found"}, 404
+
+    db.session.delete(video)
+    db.session.commit()
+
+    return jsonify({"id": video.id}), 200
+
+###Customer
```

Comment:
The basic CRUD operations for customers are largely the same as for videos, so look for places where the feedback on video endpoints could apply to customer endpoints as well.